### PR TITLE
Add "IDiagnosticProfileUAC" UAC Bypass technique

### PR DESCRIPTION
**Pull request by nasbench**
Added this technique: https://github.com/Wh04m1001/IDiagnosticProfileUAC

### CODE REVIEW

**Comment by nasbench** [comment]
According to the GitHub page, the file is required to perform a DLL sideloading/hijacking. Hence the creation of the "FileCreation" rule for "wow64log.dll"

I've searched for this file in the following repo https://github.com/AndrewRathbun/VanillaWindowsReference and didn't find a ref which means it isn't available by default on windows (at least according to the test i did)

File: rules/windows/file_event/file_event_win_uac_bypass_wow64log.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+references:
+    - https://github.com/Wh04m1001/IDiagnosticProfileUAC
+date: 2022/07/03
+logsource:
+    product: windows
+    category: file_event
+detection:
+    selection:
+        Image|endswith: '\DllHost.exe'
+        TargetFilename: 'C:\Windows\System32\wow64log.dll'
```
Comment (frack113) [inline rules/windows/file_event/file_event_win_uac_bypass_wow64log.yml:15]: wow64log.dll seen to be Redline.Stealer specific
```yaml
TargetFilename|startwith: 'C:\Windows\System32\'
TargetFilename|endswith: '.dll'
```

Comment (frack113) [reply rules/windows/file_event/file_event_win_uac_bypass_wow64log.yml:15]: Yes, when I read the references this name can very easily be change.

Comment (nasbench) [reply rules/windows/file_event/file_event_win_uac_bypass_wow64log.yml]: Done

**Comment by nasbench** [comment]
I've also added some more binaries to the rule e4a6b256-3e47-40fc-89d2-7a477edd6915

**Review by frack113 — APPROVED** [review]

**Pull request merged**